src/strategies/sonatype.js

```diff
@@ -51,18 +51,44 @@ function createPackageURLCoordinates([dependencyName, dependencyPayload]) {
   return Object.keys(versions).map((version) => toPackageURL(dependencyName, version));
 }
 
-async function fetchDataForPackageURLs(coordinates) {
+export function chunkArray(arr, chunkSize) {
+  let chunkedArr = [];
+
+  for (let i = 0; i < arr.length; i += chunkSize) {
+    chunkedArr.push(arr.slice(i, i + chunkSize));
+  }
+
+  return chunkedArr;
+}
+
+export async function fetchDataForPackageURLs(coordinates) {
+  const chunkSize = 128;
```

Comment:
You can declare this variable as a CONSTANT at the top of the code

```js
// CONSTANTS
const kChunkSize = 128;
```

Reply:
done